### DISCUSSION TITLE
More Features for NEW CONTENT area

### DISCUSSION THREAD

**Discussion by mario83s** [tidal-music/discussions#60] [Ideas & Feature requests]
I am using the "NEW CONTENT" area multiple times a week. Whenever I pick an entry the panel disappers. It would be nice to toggle a docking mode for the panel. Also the new content should support a "PLAY ALL" feaure.
![Screenshot 2024-04-25 180534](https://github.com/tidal-music/discussions/assets/98665382/02da239b-a05e-40b2-8be8-23849890657c)

**Comment by stefanmoro** [comment]
Hey @mario83s,

This GitHub discussions forum aims at handling technical discussions around the [TIDAL Developer Platform](https://developer.tidal.com/) (TIDAL API, open source projects etc).

As this question relates to an existing product feature, I will kindly ask you to reach out to our main [TIDAL support](https://support.tidal.com/).
